stable/storageos-operator/README.md

```diff
@@ -35,18 +35,21 @@ additional capacity, features and support plans contact sales@storageos.com.
     changed blocks.
 * Multiple AccessModes - dynamically provision ReadWriteOnce or ReadWriteMany
     volumes.
+* Rapid Failover - quickly detects node failure and automates recovery actions
+    without administrator intervention.
+* Data Encryption - both in transit and at rest.
```

Comment:
By reading this note only it isn't clear the reason of the separation of transit and rest. Does the data e2e encrypted, or encrypted multiple times? Maybe it is my bad, but could you please write something more straight forward?

Reply:
This may need updating to be more clear, but any change will also need to be done in the cluster-operator CSV.  I'd prefer to leave it the same as the operator for now, and we can update both separately once new text is agreed.